src/applications/hca/tests/00-all-fields.e2e.spec.js

```diff
@@ -169,9 +163,9 @@ module.exports = E2eHelpers.createE2eTest(client => {
     client.expect
```

Comment:
This it the exact line that fails in the test. This element does not exist yet because (I'm guessing) the file upload/attachment that happens on line 158 does not succeed. So this element does not get added to the DOM.